sync/bundle/message/blocks_response.go

```diff
@@ -13,16 +13,16 @@ const (
 )
 
 type BlocksResponseMessage struct {
-	ResponseCode    ResponseCode       `cbor:"1,keyasint"`
-	SessionID       int                `cbor:"2,keyasint"`
-	From            uint32             `cbor:"3,keyasint"`
-	BlocksData      [][]byte           `cbor:"4,keyasint"`
-	LastCertificate *block.Certificate `cbor:"6,keyasint"`
-	Reason          string             `cbor:"7,keyasint"`
+	ResponseCode    ResponseCode             `cbor:"1,keyasint"`
+	SessionID       int                      `cbor:"2,keyasint"`
+	From            uint32                   `cbor:"3,keyasint"`
+	BlocksData      [][]byte                 `cbor:"4,keyasint"`
+	LastCertificate *certificate.Certificate `cbor:"6,keyasint"`
```

Comment:
we missed `cbor:5`

Reply:
Good find. Thanks

Reply:
you're welcome, please fix the issue and resolve  conversation